cuvarbase/ce.py

```diff
@@ -893,11 +893,26 @@ def batched_run_const_nfreq(self, data, batch_size=10,
 
         [mem.transfer_freqs_to_gpu(**kwargs) for mem in memory]
 
+        periods = 1.0/freqs
+        periods_best, significances = [], []
```

Comment:
I'd like to leave everything consistent; since we're using frequencies everywhere else, I'll leave this as `best_freqs` and `best_freq_significances` instead of periods.